language_service/src/rename.rs

```diff
@@ -63,9 +63,11 @@ impl<'a> Rename<'a> {
     }
 
     fn get_spans_for_item_rename(&mut self, item_id: &hir::ItemId, ast_name: &ast::Ident) {
```

Comment:
Bug: Through testing we discovered that we return the wrong span here when the rename is across cells.

Reply:
Fixed